enftun/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM debian:stretch
+
+MAINTAINER Xaptum
+
+LABEL version="1.0"
+LABEL description="Xaptum base image for access to the ENF"
+
+ENV PROXY_URL http://cdn-fastly.deb.debian.org
+
+# make Apt non-interactive
+ARG DEBIAN_FRONTEND=noninteractive
+RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90-docker      && \
+    echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90-docker
+
+# Make apt-key work in Docker
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+RUN sed -i "s%{GPG_EXE}\")' --%{GPG_EXE}\")' --batch --%g" /usr/bin/apt-key
+
+# Install the Xaptum APT GPG key
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends      \
+      gnupg                                         \
+      dirmngr                                    && \
+    apt-key adv --keyserver keyserver.ubuntu.com    \
+                --recv-keys c615bfaa7fe1b4ca     && \
+    apt-get remove -y                               \
+      gnupg                                         \
+      dirmngr                                    && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install the Xaptum APT repo
+COPY xaptum.list /etc/apt/sources.list.d/xaptum.list
+
+# Install required packages
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends      \
+      enftun                                        \
+      systemd                                       \
```

Comment:
I don't believe that `systemd` is necessary.

Reply:
Ah, yep, sorry, leftover from when I was attempting to control the enftun via its systemd service.

I'll remove it.